test/ttmlir/Silicon/TTNN/simple_broadcast.mlir

```diff
@@ -1,14 +1,14 @@
 // RUN: ttmlir-opt --ttir-to-ttnn-backend-pipeline="system-desc-path=%system_desc_path%" %s > %t.mlir
 // RUN: FileCheck %s --input-file=%t.mlir
 // RUN: ttmlir-translate --ttnn-to-flatbuffer %t.mlir > %t.ttnn
-#any_device = #tt.operand_constraint<dram|l1|tile|any_device|any_device_tile>
+#any_device_tile = #tt.operand_constraint<dram|l1|tile|none|interleaved|single_bank|height_sharded|width_sharded|block_sharded|any_layout|any_device_tile>
 
-func.func public @broadcast() -> (tensor<32xf32>) {
```

Comment:
can you leave both of these test cases in repo?
The one you removed produced non valid IR which we completely ignore in our verifiers.
We have task to enrich verifiers so we could be sure that we are always producing valid IR
#1275

Reply:
@mtopalovicTT The simple_broadcast.mlir test is also in Dialect and as @jnie-TT suggested, we should be able to verification in Dialect so I moved these cases to Dialect test so it doesn't cause any issues for the device. Does that work for you for verification?